content/faq/docker/docker-context.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Docker context
+showMetadata: true
+editable: true
+showToc: true
+---
+
```

Comment:
Could you please a little introduction here to explain what is docker context?
Thanks.